packages/@webex/plugin-meetings/src/reachability/index.ts

```diff
@@ -302,8 +302,8 @@ export default class Reachability {
    * @private
    * @memberof Reachability
    */
-  private getUnreachableClusters(): Array<{name: string; protocol: string}> {
-    const unreachableList = [];
+  private getUnreachableClusters(): {name: string; protocol: string}[] {
+    const unreachableList: {name: string; protocol: string}[] = [];
```

Comment:
Just to align types, I used the more commonly used type definition for arrays [] everywhere I could and suggest preferring one style for the entire codebase.

Reply:
can this be enforced automatically with eslint? otherwise it will keep coming up

BTW, personally I prefer to use the `Array<>` syntax if the elements of the array have a long complicated type like an object with multiple properties, because it's easier to miss the `[]` at the end when reading it, but I don't mind which syntax we enforce.